content/en/features/resource-aggregate.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'Resource aggregate'
```

Comment:
Resource Aggregate is not a feature. Overview neither :)
title: 'Device Shadow'
description: 'Understand and use the device shadow'